tests/flow.html

```diff
@@ -5,7 +5,7 @@
   <title>VexFlow 2 - JavaScript Music Notation and Guitar Tab</title>
   <link rel="stylesheet" href="flow.css" type="text/css" media="screen" />
   <!-- VexFlow Sources -->
-  <script src="../build/vexflow-debug.js"></script>
+  <script src="../releases/vexflow-debug.js"></script>
```

Comment:
Yes - it seems that the built vexflox-debug.js has move so I have updated the src...flow.html stopped working without this change - did I miss something perhaps?